src/main/scala/io/archivesunleashed/spark/pythonhelpers/RecordLoaderPythonHelper.scala

```diff
@@ -0,0 +1,45 @@
+package io.archivesunleashed.spark.pythonhelpers
+
+import io.archivesunleashed.io.GenericArchiveRecordWritable.ArchiveFormat
+import io.archivesunleashed.io.{ArcRecordWritable, GenericArchiveRecordWritable, WarcRecordWritable}
+import io.archivesunleashed.mapreduce.{WacArcInputFormat, WacGenericInputFormat, WacWarcInputFormat}
+import io.archivesunleashed.spark.archive.io.{ArcRecord, ArchiveRecord, GenericArchiveRecord, WarcRecord}
+import org.apache.hadoop.io.LongWritable
+import org.apache.spark.api.java.{JavaRDD, JavaSparkContext, JavaPairRDD}
+import org.apache.spark.{SerializableWritable, SparkContext}
+import org.apache.spark.rdd.RDD
+import org.json4s.JValue
+import org.json4s.jackson.JsonMethods.parse
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.DataFrame
+import io.archivesunleashed.spark.matchbox.RecordLoader
+import io.archivesunleashed.spark.rdd.RecordRDD._
+
+object RecordLoaderPythonHelper {
+
+  def loadArc(path: String, jssc: JavaSparkContext, spark: SparkSession): DataFrame = {
+    val sc = jssc.sc
+    val rdd = RecordLoader.loadArc(path, sc).keepValidPages()
+    val df = spark.createDataFrame(rdd = rdd, beanClass = classOf[ArchiveRecord])
+    df
+  }
+
+  def loadWarc(path: String, jssc: JavaSparkContext, spark: SparkSession): DataFrame = {
+    val sc = jssc.sc
+    val rdd = RecordLoader.loadWarc(path, sc).keepValidPages()
+    val df = spark.createDataFrame(rdd = rdd, beanClass = classOf[ArchiveRecord])
+    df
+  }
+
+  def loadArchives(path: String, jssc: JavaSparkContext, spark: SparkSession): DataFrame = {
+    val sc = jssc.sc
+    val rdd = RecordLoader.loadArchives(path, sc).keepValidPages()
```

Comment:
Bigger discussion, but is there a rationale for .keepValidPages() inside vs outside of the loadArchives function?  In Scala, we require people use it separately from the load function.

Does it make sense to support accessing invalid pages?  If not, maybe alter the scala script to match this?

Reply:
Ping @lintool here

Reply:
I think the approach here works okay for this PR, but we should add an Issue to address the difference in approaches as it might be confusing.

Reply:
Yeah, my rationale was that `.keepValidPages()` was always being called after loading the archive, so I might as well make `RecordLoader` do it automatically. That being said, I can remove this if it's undesirable and revert to how it's done in Scala.

Reply:
Assuming @ruebot is okay with, I think it makes sense to move it to the RecordLoader as well.  I think that still means you need a invalid parameter check `(path, sc, invalid)` instead of including the .keepValidPages() call.

Reply:
Also - I suggest letting me fix the RecordLoader because it's probably going to break a whole bunch of unit tests.  I'll get on it Sat, Sun or Mon.

Reply:
I'd rather not do that... let's get @MapleOx 's PR merged as soon as reasonably possible. We can start issues on what needs to be done next, and attach directly to this PR.

Reply:
Yes.  @MapleOx did a great job, and this is more of a code refactor having nothing directly to do with the work.

I'll create an issue and branch off until the PR is merged.